vars/managerPipeline.groovy

```diff
@@ -150,6 +154,18 @@ def call(Map pipelineParams) {
                                    For example, {'batch_size': 2, 'parallel': 1}""",
                    name: 'mgmt_restore_params')
 
+            string(defaultValue: "${pipelineParams.get('mgmt_reuse_backup_snapshot', '')}",
+                   description: """Name of backup snapshot to use in Manager restore benchmark test, for example, 500gb_2t_ics.
+                                   The name provides the info about dataset size (500gb), number of tables (2) and compaction (ICS).
+                                   All snapshots are defined in the 'https://github.com/scylladb/scylla-cluster-tests/blob/master/defaults/manager_restore_benchmark_snapshots.yaml'""",
+                   name: 'mgmt_reuse_backup_snapshot')
+
+            string(defaultValue: "${pipelineParams.get('mgmt_ignore_post_restore_stress_read', 'true')}",
+                   description: """The flag to ignore the post-restore stress read in restore benchmark test.
+                                   Leave empty to NOT ignore the post-restore stress read.
```

Comment:
shouldn't be rather `skip` instead of `ignore`?

Reply:
Yeah, agree.
Also, since this parameter is used only for restore benchmark job (which is triggered only in manual mode), it makes sense to remove it from here and set in newly introduced `extra_environment_variables`.

Reply:
fixed